packages/rancher-logging/overlay/templates/loggings/rke/configmap.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.additionalLoggingSources.rke.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-rke
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Log_Level         info
```

Comment:
Should anything in this file be customizable ? Log level seems like something that should be.